app/views/searches/_jobs.mobile.haml

```diff
@@ -1,14 +1,19 @@
 #jobs.search.collapsible.collapsed.focus-on-next-sibling{ tabindex: -1 }
   .content-block-item.content-block-item-header
-
     = usajobs_logo
     %h3{ class: jobs_content_heading_css_classes }
       = job_openings_header(search.affiliate.agency)
-
-  - search.jobs.each_with_index do |job, index|
-    = render partial: 'job', locals: { affiliate: search.affiliate, job: job, position: index + 1 }
-  - if search.jobs.count > 3
-    .content-block-item.content-block-item-footer.content-trigger
-      = render partial: 'show_more_show_less'
+  - if search.jobs.present?
+    - search.jobs.each_with_index do |job, index|
+      = render partial: 'job', locals: { affiliate: search.affiliate, job: job, position: index + 1 }
+    - if search.jobs.count > 3
+      .content-block-item.content-block-item-footer.content-trigger
+        = render partial: 'show_more_show_less'
+  - else
+    #no-results
```

Comment:
Is this comment necessary, or leftover?

Reply:
That is not a comment. Thats a div id. I am following the same style we have for no results. You can look at https://github.com/GSA/search-gov/blob/master/app/views/searches/_no_results.mobile.haml. 
in haml, you use # for div ids and . for classes. See http://haml.info/tutorial.html Last paragraph has an example.

Reply:
Ah, gotcha. 👍